packages/app/components/DatePicker.tsx

```diff
@@ -65,19 +65,19 @@ export function DatePicker({
             )}
           >
             <span>{formattedDate()}</span>
-            <Icon name="caret-down" className="mr-2 h-4 w-4 text-black" />
+            <Icon name="caret-down" className="w-4 h-4 mr-2 text-black" />
           </Popover.Button>
           {open && (
             <>
-              <div className="fixed inset-0 bg-black/30" aria-hidden="true" />
+              <div className="fixed inset-0 z bg-black/30" aria-hidden="true" />
```

Comment:
```suggestion
              <div className="fixed inset-0 bg-black/30" aria-hidden="true" />
```
This z is a third wheel.

Reply:
fixed.